src/server/pps/server/s3g_sidecar_test.go

```diff
@@ -493,27 +524,38 @@ func TestFullS3(t *testing.T) {
 	})
 }
 
-func TestS3SkippedDatums(t *testing.T) {
+func TestFullS3(t *testing.T) {
 	if testing.Short() {
 		t.Skip("Skipping integration tests in short mode")
 	}
-	name := t.Name()
-
 	c, userToken, ns := initPachClient(t)
+	if userToken == "" {
+		userToken = "abc123"
+	}
+	t.Run("ProjectUnaware", func(t *testing.T) {
+		testFullS3(t, c, userToken, userToken, ns)
+	})
+	t.Run("ProjectAware", func(t *testing.T) {
```

Comment:
Should we make these ProjectAware test cases use non-default project?

Reply:
Good idea. Done.